src/test/java/com/avides/spring/rabbit/listener/RequestResponseSpringRabbitListenerTest.java

```diff
@@ -52,17 +52,18 @@ public void setup()
     @Test
     public void testHandleEventWithoutCorrelationId()
     {
-        try
-        {
-            MessageProperties messageProperties = new MessageProperties();
-            messageProperties.setReplyTo("response-queue");
-            successRabbitListener.handleEvent("", messageProperties);
-            fail("Exception expected");
-        }
-        catch (IllegalArgumentException e)
-        {
-            assertEquals("correlation_id must not be null", e.getMessage());
-        }
+        Tags tags = Tags.of(Tag.of("listener", "SuccessRequestResponseSpringRabbitListener"), Tag.of("from", "sender-app"));
```

Comment:
var, betrifft auch noch weitere Stellen in dieser Klasse

Reply:
Angepasst.